cmd/template/cluster/provider/aws.go

```diff
@@ -59,6 +59,11 @@ func WriteGSAWSTemplate(out io.Writer, config ClusterConfig) error {
 		Labels:         config.Labels,
 	}
 
+	crsConfig.ReleaseComponents, err = key.GetReleaseComponents(ctx, client.CtrlClient(), config.ReleaseVersion)
```

Comment:
The `ReleaseComponents` field was being referenced from the templates, but it was never initialized. So just initializing it with the right values is enough to fix this.